docs: unify module pages (#725)
denoland/docs: runtime/manual/tools/env_variables.md

### NO_PROXY

Indicates hosts which should bypass the proxy set in the other environment

> variables. See the [Proxies](#proxiesd) section for more information.

Author: typo?
```suggestion
variables. See the [Proxies](#proxies) section for more information.
```

### NPM_CONFIG_REGISTRY

The npm registry to use when loading modules via
[npm specifiers](../node/npm_specifiers.md)

## Proxies

Deno is able to handle network requests through a proxy server, useful for
various reasons such as security, caching, or accessing resources behind a
firewall. The runtime supports supports proxies for module downloads and the Web
standard `fetch` API.

Deno reads proxy configuration from environment variables: `HTTP_PROXY`,
`HTTPS_PROXY` and `NO_PROXY`.

> In case of Windows, if environment variables are not found, Deno falls back to

Author: ```suggestion
On Windows, if environment variables are not found, Deno falls back to
```
